src/anf-dotnetcore-dual-protocol-sdk-sample/Program.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+namespace Microsoft.Azure.Management.ANF.Samples
+{
+    using Microsoft.Azure.Management.ANF.Samples.Common;
+    using Microsoft.Azure.Management.NetApp;
+    using Microsoft.Azure.Management.NetApp.Models;
+    using Microsoft.Rest;
+    using System;
+    using System.Text;
+    using System.Threading.Tasks;
+    using static Microsoft.Azure.Management.ANF.Samples.Common.Utils;
+
+    class Program
+    {
+        //------------------------------------------IMPORTANT------------------------------------------------------------------
+        // Setting variables necessary for resources creation - change these to appropriated values related to your environment
+        // Please NOTE: Resource Group and VNETs need to be created prior to run this code
+        //----------------------------------------------------------------------------------------------------------------------
+
+        // Subscription - Change SubId below
+        const string subscriptionId = "<Subscription ID>";
+
+        const string resourceGroupName = "<Resource Group Name>";
+        const string location = "westus";
+        const string subnetId = "<Subnet ID>";
+        const string anfAccountName = "anftestaccount";
+        const string capacityPoolName = "anfprimarypool";
+        const string capacityPoolServiceLevel = "Standard";
+        const long capacitypoolSize = 4398046511104;  // 4TiB which is minimum size
+        const string anfVolumeName = "anftestvolume";
+        const long volumeSize = 107374182400;  // 100GiB - volume minimum size    
+
+        const string domainJoinUsername = "<Domain Join Username>";
+        const string domainJoinPassword = "<Domain Join Password>";
+        const string dnsList = "10.0.0.4,10.0.0.5"; // please note that this is a comma-seperated string
+        const string adFQDN = "testdomain.local";
+        const string smbServerNamePrefix = "pmcdns";
+        const string rootCACertFullFilePath = "<Root Certification Full File Path>";
+
+        // If resources should be cleaned up
+        static readonly bool shouldCleanUp = false;
+
+        private static ServiceClientCredentials Credentials { get; set; }
+
+        /// <summary>
+        /// Sample console application that changes a volume from one Capacity Pool to another in different service level tier
```

Comment:
Minor thing, but this text is from pool change sample

Reply:
copy/paste text error. fixed. thanks